config/default_records/tag_groups/005_ultima.yml

```diff
@@ -0,0 +1,199 @@
+Ultima P1:
+  visible: true
+  tags:
+    1: CAGCTCGAATGCGAT
+    2: CATGTGCAGCCATCGAT
+    3: CATCACACATGAATGAT
+    4: CTGTGTAGGCATGAT
+    5: CATGTATCCTCTGAT
+    6: CATATAGCCTATGAT
+    7: CGATTCATGCTCGAT
+    8: CACATCCTGCATGTGAT
+    9: CGCGCATCCTGCATGAT
+    10: CACTGCACGAATGAT
+    11: CACGCACTGCCAGAT
+    12: CTGCCATAGCACGAT
+    13: CAGTTGTGCTGTGAT
+    14: CATGGCGCGATCATGAT
+    15: CTTAGATATCATGAT
+    16: CATCCTGTGCGCATGAT
+    17: CACGTGGCACATGAT
+    18: CATGCGTCCTGTGAT
+    19: CAATGCTCTGCATAGAT
+    20: CTACATTGCACAGAT
+    21: CATAGCGAGCCAGAT
+    22: CGGAGTGCATGCATGAT
+    23: CATGGAGATACTGAT
+    24: CTGAGCCTGTCAGAT
+    25: CTCGAGATTGATGAT
+    26: CATCCATCTCATCTGAT
+    27: CACTGTCAGCCAGAT
+    28: CAGACTTGCTGCGAT
+    29: CATATAGCAGGAGAT
+    30: CATGTCTAAGATGAT
+    31: CATCGCAGCTGAATGAT
+    32: CTCTGTATTGCAGAT
+    33: CATCATCGATTCGAT
+    34: CTGTGAATATGCGAT
+    35: CTGAATGATCTCGAT
+    36: CAGAAGCTGCATGAGAT
+    37: CTAGATTGCGCAGAT
+    38: CGCATCCTCACAGAT
+    39: CACAACATATCAGAT
+    40: CATCCTGCATCGCAGAT
+    41: CACATCAGCTCAATGAT
+    42: CATGCATATAATGCGAT
+    43: CTCAATGCATCAGCGAT
+    44: CAATGTGTGCTAGAT
+    45: CTCAGCCTCATGCTGAT
+    46: CTCTCGCATGCAATGAT
+    47: CTAGCAGCCAGCGAT
+    48: CTGCCAGACTGTGAT
+    49: CGCACTGCTCCTGAT
+    50: CACATGGCAGCACAGAT
+    51: CACTTGCAGATCGAT
+    52: CTATGCCACAGCATGAT
+    53: CAACATCAGCATGAGAT
+    54: CTGCAGTGATTCATGAT
+    55: CAGCACCTGCATCAGAT
+    56: CTTATGCTATCAGAT
+    57: CAATGCGTGCTGCAGAT
+    58: CATCTCAGTGCAATGAT
+    59: CATGTGCTTCGCATGAT
+    60: CATGAGTGCAGCCAGAT
+    61: CTCAAGCATGTGCTGAT
+    62: CACAGTCAATGTGAT
+    63: CACACTTGATGCGAT
+    64: CATAGAGCCTCAGAT
+    65: CTTGTGTCATGAGAT
+    66: CAATGCACTCGAGAT
+    67: CTTATGAGCGCTGAT
+    68: CGTCATTGCACAGAT
+    69: CATCACTGCAACGAT
+    70: CTTGCATGCGATGCGAT
+    71: CGTGCGCAAGCAGAT
+    72: CTATCTCATAATGAT
+    73: CATGGCTATGCACTGAT
+    74: CTATGAATGAGCGAT
+    75: CTATGCACCATCGAT
+    76: CACAATGTGCGCGAT
+    77: CAATGACTATCTGAT
+    78: CAGCCGCTGCTCGAT
+    79: CGCGCGCAGAATGAT
+    80: CAGCCTCAGCGTGAT
+    81: CTTGATCATACAGAT
+    82: CTCTGCTGTGCAATGAT
+    83: CAGTGTATTGCTGAT
+    84: CTGTGCATCTGCCTGAT
+    85: CTCTATGTTGCTGAT
+    86: CAACTATCTGCAGAT
+    87: CAGATCTCATGAATGAT
+    88: CTATCATCCAGTGAT
+    89: CTGCTACAAGCAGAT
+    90: CAATATGCACGTGAT
+    91: CACTTCTGCGATGAT
+    92: CAAGCATATCTAGAT
+    93: CAATGACAGCTCGAT
+    94: CTGATGAGCTTGATGAT
+    95: CATATGACCTGAGAT
+    96: CAGCCGATATCAGAT
+
+Ultima P2:
+  visible: true
+  tags:
+    97: CAGTCAGTTGCAGAT
```

Comment:
I think these integer keys correspond to `map_id` on the `Tag` model. It makes sense to me to start from 1 for the first tag in a tag group - and looks like this is what most existing ones do. Was it intentional to start from 97? Thanks.

See
```
SELECT 
  MIN(map_id) AS first_map_id, 
  MAX(map_id) AS final_map_id, 
  MAX(map_id) - MIN(map_id) + 1 AS num_tags_assuming_sequential
FROM tags
GROUP BY tag_group_id
;
```

Reply:
Yep you are right, think its a copy paste error, thanks for spotting. Updated in: [9688d5e](https://github.com/sanger/sequencescape/pull/5137/commits/9688d5ea3b23b1d35f5c2e6fb6036e6aee8dd979)